TestTerraformAwsRdsExample/mysql UnitTest Failure

TestTerraformAwsRdsExample/mysql continues to fail 

```
[31m╵[0m[0m}
    destroy.go:11: 
        	Error Trace:	/home/circleci/project/modules/terraform/destroy.go:11
        	            				/usr/local/go/src/runtime/panic.go:523
        	            				/usr/local/go/src/testing/testing.go:999
        	            				/home/circleci/project/modules/terraform/apply.go:15
        	            				/home/circleci/project/test/terraform_aws_rds_example_test.go:108
        	Error:      	Received unexpected error:
        	            	FatalError{Underlying: error while running command: exit status 1; [31m╷[0m[0m
        	            	[31m│[0m [0m[1m[31mError: [0m[0m[1mRequired plugins are not installed[0m
        	            	[31m│[0m [0m
        	            	[31m│[0m [0m[0mThe installed provider plugins are not consistent with the packages
        	            	[31m│[0m [0mselected in the dependency lock file:
        	            	[31m│[0m [0m  - registry.terraform.io/hashicorp/aws: the cached package for registry.terraform.io/hashicorp/aws 4.67.0 (in .terraform/providers) does not match any of the checksums recorded in the dependency lock file
        	            	[31m│[0m [0m
        	            	[31m│[0m [0mTerraform uses external plugins to integrate with a variety of different
        	            	[31m│[0m [0minfrastructure services. To download the plugins required for this
        	            	[31m│[0m [0mconfiguration, run:
        	            	[31m│[0m [0m  terraform init
        	            	[31m╵[0m[0m}
        	Test:       	TestTerraformAwsRdsExample/mysql
    --- FAIL: TestTerraformAwsRdsExample/mysql (3.10s)
```

--- a/test/terraform_aws_rds_example_test.go
+++ b/test/terraform_aws_rds_example_test.go
@@ -8,6 +8,7 @@ import (
 	"github.com/gruntwork-io/terratest/modules/aws"
 	"github.com/gruntwork-io/terratest/modules/random"
 	"github.com/gruntwork-io/terratest/modules/terraform"
+	test_structure "github.com/gruntwork-io/terratest/modules/test-structure"
 	"github.com/stretchr/testify/assert"
 )
 
@@ -75,12 +76,13 @@ func TestTerraformAwsRdsExample(t *testing.T) {
 			awsRegion := aws.GetRandomStableRegion(t, nil, nil)
 			engineVersion := aws.GetValidEngineVersion(t, awsRegion, tt.engineName, tt.majorEngineVersion)
 			instanceType := aws.GetRecommendedRdsInstanceType(t, awsRegion, tt.engineName, engineVersion, []string{"db.t2.micro", "db.t3.micro", "db.t3.small"})
+			moduleFolder := test_structure.CopyTerraformFolderToTemp(t, "../", "examples/terraform-aws-rds-example")
 
 			// Construct the terraform options with default retryable errors to handle the most common retryable errors in
 			// terraform testing.
 			terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
 				// The path to where our Terraform code is located
-				TerraformDir: "../examples/terraform-aws-rds-example",
+				TerraformDir: moduleFolder,
 
 				// Variables to pass to our Terraform code using -var options
 				// "username" and "password" should not be passed from here in a production scenario.

